Reverse a linked list.cpp

```diff
@@ -0,0 +1,157 @@
+// Iterative C++ program to reverse
+// a linked list
+#include <iostream>
+using namespace std;
+ 
```

Comment:
Please add only those solutions of problem statement which are listed in readme table.